MykSuite/src/main/java/com/infomaniak/core/myksuite/ui/views/BaseMyKSuiteChipView.kt

```diff
@@ -32,8 +34,18 @@ abstract class BaseMyKSuiteChipView @JvmOverloads constructor(
     val tier: MyKSuiteTier,
 ) : AbstractComposeView(context, attrs, defStyleAttr) {
 
+    private var backgroundColor: Int = ResourcesCompat.ID_NULL
+
+    init {
+        context.obtainStyledAttributes(attrs, R.styleable.BaseMyKSuiteChipView, defStyleAttr, 0).apply {
+            backgroundColor = getInt(R.styleable.BaseMyKSuiteChipView_myKSuiteChipBackgroundColor, ResourcesCompat.ID_NULL)
+            recycle()
+        }
+    }
```

Comment:
You can use the `AttributeSet.getAttributes()` extension defined in Core, like other parts of the code to write it a bit more easily

Reply:
No because this module should not implement Core
And it was needless to redefine it for this only one usage